src/DynamoCore/Models/CustomNodeWorkspaceModel.cs

```diff
@@ -32,7 +33,7 @@ private set
 
         public CustomNodeWorkspaceModel(
             string name, string category, string description, double x, double y, Guid customNodeId,
-            NodeFactory factory, string fileName="")
+            NodeFactory factory, ElementResolver elementResolver, string fileName="")
```

Comment:
@Benglin as a response to this, please see my comments above and let me know if they make sense.